target/linux/ramips/image/mt76x8.mk

```diff
@@ -522,6 +522,14 @@ define Device/oraybox_x1
 endef
 TARGET_DEVICES += oraybox_x1
 
+define Device/qding_qc202
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := Qding
+  DEVICE_MODEL := QC202
+  DEVICE_PACKAGES := kmod-rtc-ds1307 kmod-i2c-mt7628 kmod-spi-dev
```

Comment:
What's this spi-dev for?

Reply:
I dont know, but is installed in original firmware. So I just added it. Should I remove it?

Reply:
Maybe this is how the Bluetooth chip communicates. I don't know what kmod-qd-dev in the original firmware is used for, but it looks very simple.